menas/src/main/scala/za/co/absa/enceladus/menas/services/DatasetService.scala

```diff
@@ -56,12 +59,16 @@ class DatasetService @Autowired() (datasetMongoRepository: DatasetMongoRepositor
         coordPath <- oozieRepository.createCoordinator(newDataset, wfPath)
         coordId <- latest.schedule match {
           case Some(sched) => sched.activeInstance match {
-              case Some(instance) => oozieRepository.killCoordinator(instance.coordinatorId).flatMap({ res =>
-                  oozieRepository.runCoordinator(coordPath, newDataset.schedule.get.runtimeParams)
-                })
+              case Some(instance) => 
```

Comment:
This is too much indentation for my taste. Could it be extracted without it looking worse?

Reply:
Haha yeah I thought this was gonna come up... got any suggestions?

Reply:
Oh sorry, didn't see this was extra indented. Fixed. Feel free to suggest refactoring for this btw.